src/content/whats-new/2022/02/InfraV2-WhatsNew.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 'New Infrastructure Monitoring Experience is now Generally Available'
```

Comment:
Maybe have this as sentence case - 'New infrastructure monitoring experience is now generally available'

Change title to 'Pinpoint potential infrastructure issues and quantify blast radius with a new infrastructure monitoring experience' - approved by Kevin (PMM)